x-team/xp-ui
# Add applicant screen notification bar

**Release Type:** *Non-Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3376

## Description

Adds two new components (and stories for each one - also with knobs):
- `NotificationBar`
- `ApplicantScreenNotification` (which composes `NotificationBar`

then adds the `ApplicantScreenNotification` to the `ApplicantScreen`.

## Checklist

## Steps to Test or Reproduce

- On storybook (https://deploy-preview-465--xpui-develop.netlify.com/?path=/story/screens-and-layouts-applicantscreen--basic-usage)
- observe the notification bar is present
- open the knobs tab
- toggle the know and view the notification go away and come back

## Impacted Areas in Application

`ApplicantScreen` component

## Screenshots

**ApplicantScreen:**
![image](https://user-images.githubusercontent.com/15269623/67468819-07a1fc00-f600-11e9-9985-73b38a7d4eab.png)

**ApplicantScreenNotification:**
![image](https://user-images.githubusercontent.com/15269623/67468933-4041d580-f600-11e9-927c-4ffe2d2644ac.png)

**NotificationBar:**
![image](https://user-images.githubusercontent.com/15269623/67469018-636c8500-f600-11e9-9f4a-b926108d590f.png)
